reinforcement_learning/rl_on_gcp_demo/trainer/td3_agent.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implementation of a TD3 agent.
+
+Implementation of TD3 - Twin Delayed Deep Deterministic Policy Gradient
+Algorithm and hyperparameter details can be found here:
+https://arxiv.org/pdf/1802.09477.pdf
+
+"""
+import agent
+from common import replay_buffer
+from common.actor_critic import ActorNetwork, CriticNetwork
+import numpy as np
+
+
+class TD3(agent.Agent):
+    """TD3 agent."""
+
+    def __init__(self, env, sess, config):
+        """Initialize members."""
+        self.env = env
+        state_dim = env.observation_space.shape[0]
+        self.action_dim = env.action_space.shape[0]
+        self.action_high = env.action_space.high
+        self.action_low = env.action_space.low
+        self.batch_size = config.batch_size
+        self.warmup_size = config.warmup_size
+        self.gamma = config.gamma
+        self.sigma = config.sigma
+        self.sigma_tilda = config.sigma_tilda
+        self.c = config.c
+        self.d = config.d
```

Comment:
Same as above

Reply:
fixed in ac35cd106aa30d9f4e0cc69898e7da768c8518a7